include/dgl/runtime/ndarray.h

```diff
@@ -182,6 +182,8 @@ class NDArray {
    *       IsPinned: directly return;
    *       kDGLCUDA: invalid, will throw an error.
    */
+
+  inline NDArray PinMemory();
```

Comment:
Can you add a specific comment for `PinMemory()`? For example, "Return a copy of the array in pinned memory".